feature-group-definitions/http11.yml

```
@@ -0,0 +1,76 @@
+name: HTTP/1.1
+spec: https://httpwg.org/specs/rfc9112.html
+compat_features:
+  - http.headers.Accept
+  - http.headers.Accept-Encoding
+  - http.headers.Accept-Language
+  - http.headers.Accept-Ranges
+  - http.headers.Age
+  - http.headers.Authorization
+  - http.headers.Cache-Control
+  - http.headers.Connection
+  - http.headers.Content-Encoding
+  - http.headers.Content-Language
+  - http.headers.Content-Length
+  - http.headers.Content-Location
+  - http.headers.Content-Range
+  - http.headers.Content-Type
+  - http.headers.Date
+  - http.headers.ETag
+  - http.headers.Expect
+  - http.headers.Expires
+  - http.headers.From
+  - http.headers.Host
+  - http.headers.If-Match
+  - http.headers.If-Modified-Since
+  - http.headers.If-None-Match
+  - http.headers.If-Range
+  - http.headers.If-Unmodified-Since
+  - http.headers.Last-Modified
+  - http.headers.Location
+  - http.headers.Pragma
+  - http.headers.Proxy-Authenticate
+  - http.headers.Range
+  - http.headers.Referer
+  - http.headers.Retry-After
+  - http.headers.Server
+  - http.headers.TE
+  - http.headers.Trailer
+  - http.headers.Transfer-Encoding
+  - http.headers.User-Agent
+  - http.headers.Vary
+  - http.headers.Via
+  - http.headers.Warning
+  - http.headers.WWW-Authenticate
+  - http.methods.CONNECT
+  - http.methods.DELETE
+  - http.methods.GET
+  - http.methods.HEAD
+  - http.methods.OPTIONS
+  - http.methods.POST
+  - http.methods.PUT
+  - http.methods.TRACE
+  - http.status.100
+  - http.status.200
+  - http.status.201
+  - http.status.204
+  - http.status.206
+  - http.status.301
+  - http.status.302
+  - http.status.303
+  - http.status.304
+  - http.status.307
+  - http.status.401
+  - http.status.403
+  - http.status.404
+  - http.status.406
+  - http.status.407
+  - http.status.409
+  - http.status.410
+  - http.status.412
+  - http.status.416
```

```suggestion
  - http.status.416
  - http.status.418
```

This is a haha only serious suggestion. There are a few other >416 statuses. Should they be here?

---

https://datatracker.ietf.org/doc/html/rfc2616 only lists 417 Expectation Failed, but https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/417 doesn't have a compat table. If there was a BCD entry we should map it to HTTP/1.1. The statuses 418, 425, and 451 aren't in https://datatracker.ietf.org/doc/html/rfc2616.